graph-builder/src/graph.rs

```diff
@@ -98,6 +100,8 @@ pub async fn index(
     req: HttpRequest,
     app_data: actix_web::web::Data<State>,
 ) -> Result<HttpResponse, GraphError> {
+    let _span = get_tracer().start("index", None);
```

Comment:
Would

```suggestion
    let _ = get_tracer().start("index", None);
```

work as well?